k8s/autopilot/autopilotruleobject.go

```diff
@@ -0,0 +1,59 @@
+package autopilot
+
+import (
+	autv1alpha1 "github.com/libopenstorage/autopilot-api/pkg/apis/autopilot/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// RuleObjectOps is an interface to perform k8s AutopilotRuleObjects operations
+type RuleObjectOps interface {
+	// CreateAutopilotRuleObject creates the AutopilotRuleObject object
+	CreateAutopilotRuleObject(object *autv1alpha1.AutopilotRuleObject) (*autv1alpha1.AutopilotRuleObject, error)
+	// GetAutopilotRuleObject gets the AutopilotRuleObject for the provided name
+	GetAutopilotRuleObject(string) (*autv1alpha1.AutopilotRuleObject, error)
```

Comment:
AROs are now namespace scoped. So all APIs need the namespace field.

They namespace scoped since this commit: https://github.com/libopenstorage/autopilot-api/commit/2e4981985017f4ba3676ae2c6b585b3cfc756d0f

Reply:
fixed